Userland/Libraries/LibCore/EventLoop.h

```diff
@@ -62,8 +62,10 @@ class EventLoop {
 
     bool was_exit_requested() const { return m_exit_requested; }
 
-    static int register_timer(Object&, int milliseconds, bool should_reload, TimerShouldFireWhenNotVisible);
-    static bool unregister_timer(int timer_id);
+    using TimerId = int;
+    static constexpr TimerId kInvalidTimerId = 0;
```

Comment:
We don't use this naming style, please adhere to https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md#names